drivers/hmis/lib/tasks/ce_starter_pack_20250302.rake

```diff
@@ -48,7 +48,139 @@ task ce_starter_pack_20250302: [:environment] do
+      message: 'start_referral',
+    },
+  ]
+  start_workflow_event.save!
+
+  accept_workflow_event = Hmis::WorkflowDefinition::EndEvent.find_or_initialize_by(
+    name: 'accept referral',
+    template_id: one_task_template.id,
+  )
+  accept_workflow_event.swimlane_id = case_managers.id
+  accept_workflow_event.trigger_config = [
+    {
+      event: 'end_workflow',
+      message: 'accept_referral',
+    },
+  ]
+  accept_workflow_event.save!
+
+  reject_workflow_event = Hmis::WorkflowDefinition::EndEvent.find_or_initialize_by(
+    name: 'reject referral',
+    template_id: one_task_template.id,
+    )
+  reject_workflow_event.swimlane_id = case_managers.id
+  reject_workflow_event.trigger_config = [
+    {
+      event: 'end_workflow',
+      message: 'reject_referral',
+    },
+  ]
+  reject_workflow_event.save!
+
+  form_definition = Hmis::Form::Definition.find_or_initialize_by(
+    identifier: 'confirm_client_accepts_referral',
+    status: 'published'
+  )
+  form_definition.title = 'Confirm Client Accepts Referral'
+  form_definition.role = 'CE_REFERRAL_STEP'
+  form_definition.version ||= 0
+  form_definition.definition = {
+    "item": [
+      {
+        "text": "Date Contacted",
+        "type": "DATE",
+        "link_id": "date_contacted",
+        "required": true,
+        "read_only": false,
+        "warn_if_empty": false,
+        "disabled_display": "HIDDEN",
+        "mapping": {"custom_field_key": "ce_one_task_date_contacted"}
+      },
+      {
+        "text": "Client Accepts Referral",
+        "type": "CHOICE",
+        "link_id": "client_accepted",
+        "required": true,
+        "read_only": false,
+        "warn_if_empty": false,
+        "disabled_display": "HIDDEN",
+        "pick_list_options": [
+          {
+            "code": "1",
+            "label": "Yes, client accepts referral"
+          },
+          {
+            "code": "0",
+            "label": "No, client does not accept referral or could not be contacted"
+          }
+        ],
+        "mapping": {"custom_field_key": "ce_one_task_client_accepted"}
+      },
+      {
+        "text": "If you submit, the referral will be stopped and sent to an Admin for review. This cannot be undone.",
+        "type": "DISPLAY",
+        "link_id": "stop_referral_if_you_submit_the_referral",
+        "component": "ALERT_ERROR",
+        "enable_when": [
+          {
+            "operator": "EQUAL",
+            "question": "client_accepted",
+            "answer_code": "0"
+          }
+        ],
+        "enable_behavior": "ALL",
+        "disabled_display": "HIDDEN"
+      },
+      {
+        "text": "Notes",
+        "type": "TEXT",
+        "link_id": "notes",
+        "disabled_display": "HIDDEN",
+        "mapping": {"custom_field_key": "ce_one_task_notes"}
+      }
+    ]
+  }
+  form_definition.save! if form_definition.changed?
+  # TODO - add CDEDs? Not necessary/used for CE, but the form builder prevents edits if it can't find them
```

Comment:
If these forms are created using the editor, then they will have (unused) cdeds created for them on publish anyway.

I guess we could either:
- create some cdeds here in the starter pack, just to help down the road with any form builder edits
- update the form builder to make an exception for certain form types that don't need cdeds

Reply:
It's a good question; I wonder if there's a reason we might need cdeds. If not it would be cleaner to make them optional

Reply:
@gigxz @ttoomey Relatedly, have we discussed using Form Processors for ce steps? Does it sound like too much effort? I haven't played around with it and don't really have a clear idea of how it would work exactly. But the CE workflow engine will eventually create enrollments and CE events, sort of reminiscent of the Form Processor's "related records"

Reply:
I would like to discuss it more, particularly re: reporting needs for CE. (Responses to CE referral task fields will need to be reportable in OP Analytics).

For the related record changes, I think we agreed that it would be sufficient to have a fixed set of "side effects" that CE forms could perform (e.g. create Enrollment, set Move-in Date, create CE Event, set CE Event result..), a.k.a. it's not necessary to make it so flexible that it's able to update _any_ field on Enrollment (for example).

Reply:
Ok, sounds good, let's discuss more. I think for this PR I'd like to leave this as-is. I'll split out a separate ticket to fully enable editing form definitions for CE using the form builder, since I think there's more discussion needed there.
- Decision about CDEDs - should CE forms create them?
  - If yes:  add them here in the starter pack; 
  - If no: prevent the form builder from failing if it can't find them
- (proposed, needs discussion) Link CE workflow node to a form _identifier_, not a form definition